### PR TITLE
2018 Joomla CMS Security Vulnerabilities.

### PR DESCRIPTION
**Pull request by Nenglish7**
- https://github.com/joomla/joomla-cms

### CODE REVIEW

**Comment by Nenglish7** [comment]
I am confused on the name of the branch. The actual branch name is `staging` but it's invalid.

File: joomla/joomla-cms/CVE-2018-6376.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title: SQLi vulnerability in Hathor postinstall message
+link: https://developer.joomla.org/security-centre/722-20180105-core-sqli-vulnerability.html
+cve: CVE-2018-6376
+branches:
+    master:
+        time:     2018-1-30
+        versions: ['>=3.7.0', '<3.8.3']
+reference: composer://joomla/joomla-cms
```
Comment (cebe) [inline joomla/joomla-cms/CVE-2018-6376.yaml]: `joomla/joomla-cms` does not seem to be a package available on packagist. how is it distributed?

File: joomla/joomla-cms/CVE-2018-6376.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title: SQLi vulnerability in Hathor postinstall message
+link: https://developer.joomla.org/security-centre/722-20180105-core-sqli-vulnerability.html
+cve: CVE-2018-6376
+branches:
+    master:
+        time:     2018-1-30
```
Comment (stof) [inline joomla/joomla-cms/CVE-2018-6376.yaml]: please use `01` for the month

**Comment by stof** [comment]
@Nenglish7 any chance to answer the comments here ?

File: joomla/joomla-cms/CVE-2018-6376.yaml (hunk 1 above)
Comment (stof) [reply joomla/joomla-cms/CVE-2018-6376.yaml]: `joomla/joomla-cms` is indeed not registered on Packagist, as it is a project using Composer for its own deps, not something you can install as a dependency.

File: joomla/joomla-cms/CVE-2018-6376.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+        versions: ['>=3.7.0', '<3.8.3']
```
Comment (stof) [inline joomla/joomla-cms/CVE-2018-6376.yaml]: this version range is wrong. The link you gave says that 3.8.3 is also vulnerable, and the fix is part of 3.8.4.

I haven't checked all other ones

**Comment by fabpot** [comment]
Closing as there is no feedback.

**Pull request closed without merging**